app/models/counselor.rb

```diff
@@ -4,4 +4,6 @@ class Counselor < ApplicationRecord
   has_many :counselors_groups
   has_many :groups, through: :counselors_groups
 
+  validates :alias, presence: true
```

Comment:
This may break something. I remember we set the default alias to blank (it was "none" but I asked you to change it).